README.md

```diff
@@ -65,6 +65,9 @@ Some playbooks require third-party roles, which are specified in `bin/requiremen
 $ bin/setup
 ```
 
+## Deployment with Ansiable
```

Comment:
```suggestion
## Deployment with Ansible
```

Reply:
I did not spot that one :sweat_smile:  I've taken the name from [here](https://github.com/openfoodfoundation/ofn-install/wiki/Deployment-with-Ansiable) -> I've fixed it it now on the wiki, as well.

Reply:
I think it should be 'Ansible' (https://www.ansible.com/)

Reply:
> I think it should be 'Ansible' (https://www.ansible.com/)

Of course... Sorry, for my mix up.